keyboards/keebfront/vanguard65/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency
+F_CPU = 8000000
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+LTO_ENABLE = yes
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes      # Mouse keys 
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = yes          # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+ENCODER_ENABLE = yes	   # Enable encoder
```

```suggestion
ENCODER_ENABLE = yes       # Enable encoder
```
Also, if encoders are enabled at the keyboard level, there should be some encoder functionality at the keyboard level as well using the `encoder_update_kb()` function in `vanguard65.c` file, that way it works for QMK Configurator users as well.

---

Added encoder map to latest default keymap.c